src/std/main.ab

```diff
@@ -111,6 +111,34 @@ pub fun sum(list: [Num]): Num {
     return unsafe $echo "{list}" | awk '\{s=0; for (i=1; i<=NF; i++) s+=\$i; print s}'$ as Num
 }
 
+pub fun array_get(array, index) {
+    return array[index]
+}
```

Comment:
I advise to remove this function. It'll get added promptly as it's an important feature

```suggestion
```

Reply:
issue is #211 
if #211 is resolved ``array_get`` is not necessary

Reply:
Oh yeah thanks @CymDeveloppement!

Reply:
I removed the ``array_get`` function